TypeScript.YAML-tmLanguage

```diff
@@ -37,15 +37,15 @@ repository:
     end: (?=$|[;,])
     patterns:
     - include: '#ternary-operator'
-    - include: '#ternary-expression-type' 
+    - include: '#expression-type' 
 
   ternary-operator:
     begin: (\?)
     end: (:)
     patterns:
-    - include: '#ternary-expression-type' 
+    - include: '#expression-type' 
 
-  ternary-expression-type:
+  expression-type:
```

Comment:
This is only used in the true/false branch of a ternary expression, right? How about `ternary-expression-branch`?